scripts/artifactory.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -e
```

Comment:
I believe this can be inlined: `#!/usr/bin/env bash -e`.

Reply:
It didn't like that :man_shrugging: